test/optimizations/forall-unordered-opt/compilation-tests/blocking-analysis1.chpl

```diff
@@ -1,3 +1,4 @@
+use CPtr;
```

Comment:
It looks to me like this change (and the one to the .good) shouldn't be required.